workflow/rules/annotation/pass1.smk

```diff
@@ -1,6 +1,18 @@
+rule download_snpeff_db:
+    output:
+        directory("{working_dir}/results/databases/{{ref}}".format(working_dir=os.getcwd()))
```

Comment:
The `{working_dir}` is not necessary. Just have a relative path starting with results. It will always be in the working dir. If snpeff expects an absolute path, rather have this translated in params.

Reply:
Right, it is only necessary for the snpeff call and I just translated this syntax to the output.